packages/flutter_markdown/lib/src/widget.dart

```diff
@@ -12,6 +12,17 @@ import 'package:markdown/markdown.dart' as md;
 import '../flutter_markdown.dart';
 import '_functions_io.dart' if (dart.library.html) '_functions_web.dart';
 
+/// Signature for callbacks used by [MarkdownWidget] when
+/// [MarkdownWidget.selectable] is set to true and the user changes selection.
+/// The callback will return the entire block of text available for selection,
```

Comment:
There should be a line separating the first sentence and the rest of the comment.